user_tools/src/spark_rapids_pytools/resources/dev/prepackage_mgr.py

```diff
@@ -87,8 +86,7 @@ def __init__(self,
             self.dest_dir = FSUtil.get_abs_path(self.dest_dir)
 
     def _get_spark_rapids_jar_url(self) -> str:
-        jar_version = Utils.get_latest_available_jar_version(self._mvn_base_url,  # pylint: disable=no-member
-                                                             Utils.get_base_release())
+        jar_version = Utilities.get_latest_mvn_jar_from_metadata(self._mvn_base_url)  # pylint: disable=no-member
```

Comment:
Curious on the change of package here. Why is that needed?

Reply:
That's a good question.
Few months ago, I did the implementation of the new simplified CLI "spark_rapids", and the implementation is located under `spark_rapids_tools` package.
Ideally, we should use the functionality in the new package but this will be a little bit invasive step.
So, I try my best to incrementally migrate some functionality when it is applicable. For example here, since I want a new implementation to parse the XML, it is a good opportunity to move that into the new package.